services/api/src/configuration/index.ts

```diff
@@ -54,6 +54,14 @@ export const Configuration = {
     return env.get('ALGOD_PORT').default('4001').asPortNumber()
   },
 
+  get algodEnv() {
+    return env
+      .get('ALGOD_ENV')
+      .required()
+      .default('testnet')
+      .asEnum(['betanet', 'testnet', 'mainnet'])
+  },
```

Comment:
As defined in `.env.example`, this value can be used to determine which flavor of the blockchain we want to interface with. In this PR, this determines which API endpoint to hit on AlgoExplorer, but could also see this being used for the wallet SDKs? 

Is the name `ALGOD_ENV` okay?